tests/unit_tests/test_wave_06.py

```diff
@@ -228,21 +229,22 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert not result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_c]
+    assert jesse.inventory == [item_d, item_e, item_f]
```

Comment:
Do we need to consider order when checking the inventories of these vendors? If not, we don't need to test for it here. 

Same comment applies to similar assertions written in another test below.